apps/emqx_auth_http/src/emqx_auth_http_app.erl

```diff
@@ -125,17 +125,17 @@ load_hooks() ->
             ok = emqx_acl_http:register_metrics(),
             PoolOpts2 = proplists:get_value(pool_opts, ACLReq),
             PoolName2 = proplists:get_value(pool_name, ACLReq),
-            ehttpc_sup:start_pool(PoolName2, PoolOpts2),
+            {ok, _} = ehttpc_sup:start_pool(PoolName2, PoolOpts2),
             emqx:hook('client.check_acl', {emqx_acl_http, check_acl, [#{acl => maps:from_list(ACLReq)}]})
     end,
     ok.
 
 unload_hooks() ->
     emqx:unhook('client.authenticate', {emqx_auth_http, check}),
     emqx:unhook('client.check_acl', {emqx_acl_http, check_acl}),
-    ehttpc_sup:stop_pool('emqx_auth_http/auth_req'),
-    ehttpc_sup:stop_pool('emqx_auth_http/super_req'),
-    ehttpc_sup:stop_pool('emqx_auth_http/acl_req'),
+    ok = ehttpc_sup:stop_pool('emqx_auth_http/auth_req'),
```

Comment:
it is perhaps too defensive to match `ok` when unloading hooks.

Reply:
change to `_` ?

Reply:
fixed